app/assets/javascripts/locations.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
This is automatically generated, yeah? We're not planning on actually using coffeescript in this file, are we? :/